vnext/Microsoft.ReactNative/Fabric/Composition/CompositionRootAutomationProvider.cpp

```diff
@@ -1,6 +1,13 @@
 #include "pch.h"
 #include "CompositionRootAutomationProvider.h"
+#include <algorithm>
 #include "UiaHelpers.h"
+#pragma warning(push)
+#pragma warning(disable : 4229)
+#define IN
+#define OUT
```

Comment:
What's this for? `SAFEARRAY`?
Most of the code is using std::vector, so where's the SAFEARRAY dependency being forced?

Reply:
yea, a lot of the UIA APIs use SAFEARRAY for certain parameters.